docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
+- Docker version v(=docker.version=) or later.
+- A PostgresSQL database server v14 or later.
```

Comment:
I'd add that RDS is also supported.

Reply:
Good point. We also have IAM auth support, I can try to document it (+ validate since it has sat unused for a bit).